src/CollectionMacroServiceProvider.php

```diff
@@ -19,5 +21,11 @@ public function register()
             ->each(function ($macro, $path) {
                 require_once $path;
             });
+
+        /*
+         * Research
+         */
+        Collection::mixin(new AtMacro());
```

Comment:
Let's use `Collection::macro('at', new AtMacro());` here.

Reply:
Sure thing! 👍 

The tweet you mention in #154 does not show implementation..

Can you help me please, pointing how this Macro class should contain??

Constructors may not return, so I'm fiddling around with `__invoke` but getting frustrated because of `this` context..